src/librustc_typeck/diagnostics.rs

```diff
@@ -1550,6 +1550,30 @@ impl Foo for Bar {
 ```
 "##,
 
+E0191: r##"
```

Comment:
You may want to make the reason clearer (trait objects need to have all associated types specified).